### PR TITLE
add unit tests for `sampling_bias_manual`

### PR DESCRIPTION
**Pull request by sangovae**
- some unit tests, not finished yet
- also improved styling of the cli checks

### CODE REVIEW

File: tests/testthat/test-sampling_bias_manual.R [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+# Unit tests
+## expect errors
+test_that("arguments are of the right class", {
+  # occurrences_sf is an sf object
+  expect_error(
+    sampling_bias_manual(
+      occurrences_sf = occurrences,
+      bias_weights = bias_weights01_sf
+    ),
+    class = "gcube_error_class_occurrences_sf"
```
Comment (PietrH) [inline tests/testthat/test-sampling_bias_manual.R:53]: Apart from the class you can also test the exact error message if you want.

File: tests/testthat/test-sampling_bias_manual.R [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+    sampling_bias_manual(
+      occurrences_sf = points_sf1,
+      bias_weights = grid_sf_withoutweights
+    ),
+    class = "gcube_error_column_names"
+  )
+  }
+)
+
+## expect right output format
```
Comment (PietrH) [inline tests/testthat/test-sampling_bias_manual.R:97]: I added some tests regarding the returned object, feel free to make any changes you like

File: R/sampling_bias_manual.R [hunk 3]
```diff
@@ -68,67 +68,89 @@ sampling_bias_manual <- function(occurrences_sf, bias_weights) {
...
+      ),
+      class = "gcube_error_column_names"
+    )
   }
 
   # Check if crs is the same
   if (sf::st_crs(occurrences_sf) != sf::st_crs(bias_weights)) {
-    cli::cli_abort("sf::st_crs(occurrences_sf) == sf::st_crs(bias_weights) is not TRUE")
+    cli::cli_abort(
+      "sf::st_crs(observations) == sf::st_crs(grid) is not TRUE"
```
Comment (PietrH) [inline R/sampling_bias_manual.R:105]: This isn't a very clear error message to me, what does `st_crs(grid) == TRUE` actually mean? 

Something like occurrences_sf and bias_weights have a different crs, occurrences_sf has crs: ... and bias_weights has crs : ... but they need to be the same. But maybe you can word it better than I can.

**Review by sunray1 — APPROVED** [review]

**Comment by sunray1** [comment]
Great job!

**Review by PietrH — APPROVED** [review]
I see you were still working on this when you made the PR, but I had a look at it already anyway. 

It looks good! I made some small notes and additions. You seem to really be getting the hang of unit testing these error messages :)

A small general tip to unit testing is to think about what user behaviour could go wrong. What kind of arguments could they use, in what combination. What happens if you forget one, or if something is wrong with one.. The first step is of course to make sure the function works if the user acts as you expect (like the tests I added). 

Well done!

**Pull request merged**